test/test_jit.py

```diff
@@ -6851,6 +6851,17 @@ def foo(a, b):
                     with self.assertRaisesRegex(RuntimeError, 'division by 0'):
                         foo(i, j)
 
+    # Testing bitwise shorthand aug assignment
+    def test_bool_augassign(self):
+        @torch.jit.script
+        def func(a : bool, b : bool):
```

Comment:
```suggestion
        def func(a: bool, b: bool) -> bool:
```

Reply:
Also, what about augmented assignment with other bitwise ops? --> Should I handle all the ops in this same PR? or do you want me to do it in a different one?

Reply:
Either one is fine, but it's just that the PR title and message imply that all of them are handled. So you can either all of them here or make a stack of 3-4 PRs, one for each op.